app/assets/javascripts/calendars/appointment-availability.es6

```diff
@@ -59,19 +59,46 @@
 
       super(el, calendarConfig);
 
+      this.coloursConfig = [
+        { count: 30, colour: 'hsl(200, 50%, 25%)' },
+        { count: 20, colour: 'hsl(200, 50%, 35%)' },
+        { count: 10, colour: 'hsl(200, 50%, 45%)' },
+        { count: 5,  colour: 'hsl(200, 50%, 55%)' },
+        { count: 1,  colour: 'hsl(200, 50%, 65%)' }
+      ];
+
       this.init();
       this.addEvents();
     }
 
+    getEventColour(event) {
+      if (event.selected === true) {
+        return this.selectedEventColour;
+      }
+
+      const colours = $.grep(this.coloursConfig, (configItem) => {
```

Comment:
Think we'll have to play it by ear with the total - at the moment they have an overlap of 15 guiders in the early shift with 15 in the mid shift that do a 12.20 appointment, making up to 30 guiders at one time available - if they do have more guiders available at one particular time it just means that slot will be dark blue.

`find` is used to filter DOM elements, rather than `$.grep()` which is used to filter an array.